app/assets/stylesheets/ama_layout/layout/sidebar.scss

```diff
@@ -1,104 +1,108 @@
-
 .dashboard-title {
   font-size: $base-font-size*0.875;
   font-weight: normal;
-  color: #5C5C5C;
+  color: $charcoal;
   text-transform: uppercase;
-  background:white;
-  padding: $base-margin*3;
+  background: $white;
+  padding: $base-padding*3;
 }
 
-.dashboard-nav {
+.dashboard-nav{
```

Comment:
@MathieuGilbert Just FYI, Kayt and I decided that as part of our code standard we don't want a space there.